api/src/pcapi/routes/backoffice/bookings/forms.py

```diff
@@ -273,3 +273,7 @@ class CancelIndividualBookingForm(FlaskForm):
 
 class BatchCancelIndividualBookingsForm(BatchForm, CancelIndividualBookingForm):
     pass
+
+
+class BatchTagFraudulentBookingsForms(BatchForm):
+    send_mails = fields.PCSwitchBooleanField("Envoyer un mail d'avertissement aux acteurs culturels")
```

Comment:
Le rendu n'est pas très heureux : 
![image](https://github.com/user-attachments/assets/3e2a908e-add0-435a-bf9e-62a6cfff5b40)

Comme les _switches_ sont par défaut deux par lignes, tu peux facilement corriger : 
```suggestion
    send_mails = fields.PCSwitchBooleanField("Envoyer un mail d'avertissement aux acteurs culturels", full_row=True)
```

Reply:
Ah, il me semblait que c'était un bug plus général de nos switchs, car ce n'est pas le seul à en souffrir; merci pour l'info :)